### PR TITLE
Minor libkcapi test improvements

### PR DESCRIPTION
**Pull request by WOnder93**
Explanations are in the commit messages.

### CODE REVIEW

File: packages/libkcapi/af_alg-test/runtest.sh [hunk 1]
```diff
@@ -29,8 +29,6 @@ rlJournalStart
         rlRun "git clone '$GIT_URL' libkcapi"
         rlRun "(cd libkcapi && git checkout $GIT_REF)"
         rlRun "(cd libkcapi && autoreconf -i)"
```
Comment (emanchado) [inline packages/libkcapi/af_alg-test/runtest.sh:31]: The `autoreconf` line is still needed?

Comment (WOnder93) [reply packages/libkcapi/af_alg-test/runtest.sh:31]: Yes, the `test-invocation.sh` script needs the `configure` script to be generated already.

**Comment by emanchado** [comment]
Should I merge, or do you want a second review or something?

**Comment by WOnder93** [comment]
You can merge it, as far as I'm concerned. Thanks for the review!

**Pull request merged**